packages/console/src/pages/JwtClaims/utils/config.tsx

```diff
@@ -55,10 +55,12 @@ declare global {
      * This function is called to get custom claims for the JWT token.
      * 
      * @param {${JwtCustomizerTypeDefinitionKey.ClientCredentialsPayload}} token -The JWT token.
+     * @param {undefined} data - Logto internal data placeholder that can be used to pass additional information (undefined for M2M accesstoken)
+     * @param {${JwtCustomizerTypeDefinitionKey.EnvironmentVariables}} envVariables - The environment variables.
      * 
      * @returns The custom claims.
      */
-    getCustomJwtClaims: (token: ${JwtCustomizerTypeDefinitionKey.ClientCredentialsPayload}, envVariables: ${JwtCustomizerTypeDefinitionKey.EnvironmentVariables}) => Promise<CustomJwtClaims>;
+    getCustomJwtClaims: (token: ${JwtCustomizerTypeDefinitionKey.ClientCredentialsPayload}, data: undefined, envVariables: ${JwtCustomizerTypeDefinitionKey.EnvironmentVariables}) => Promise<CustomJwtClaims>;
```

Comment:
if it's always undefined, why would we need it anyway?

Reply:
Need to align the function's interface definition under access token and client credentials use cases.